packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/tests/index.test.js

```diff
@@ -878,8 +878,8 @@ describe('ADMIN | Pages | API TOKENS | ListPage', () => {
                           class="c5 c31"
                         >
                           <time
-                            datetime="2021-11-15T00:00:00.000Z"
-                            title="11/15/2021 12:00 AM"
```

Comment:
@simotae14 Do you understand why these have changed? I don't think we should commit this if there is no good reason and I believe it needs to be understood.

Reply:
they are the same but they failed, not just this one, other snapshots tests, I will revert back the code after this attempt